### PR TITLE
New feature: cpu burst strategy

### PR DESCRIPTION
**Pull request by stormgbs**
Signed-off-by: stormgbs <stormgbs@gmail.com>

### Ⅰ. Describe what this PR does

New feature: cpu burst strategy

### Ⅱ. Does this pull request fix one issue?
#52

### CODE REVIEW

**Comment by jasonliu747** [comment]
@stormgbs Hi, I have skipped the unit test `Test_metaService_syncPods` in #80. Please rebase your code and push it again. Sorry for any inconvenience.

File: pkg/koordlet/resmanager/cpu_burst.go [hunk 1]
```diff
@@ -0,0 +1,602 @@
+/*
```
Comment (jasonliu747) [inline pkg/koordlet/resmanager/cpu_burst.go:1]: Add UT.

File: pkg/koordlet/resmanager/resource_update_executor.go [hunk 2]
```diff
@@ -0,0 +1,111 @@
+/*
```
Comment (jasonliu747) [inline pkg/koordlet/resmanager/resource_update_executor.go:1]: Add UT.

File: pkg/koordlet/resmanager/types.go [hunk 3]
```diff
@@ -0,0 +1,301 @@
+/*
```
Comment (jasonliu747) [inline pkg/koordlet/resmanager/types.go:1]: Add UT.

File: pkg/koordlet/resmanager/types.go [hunk 4]
```diff
@@ -0,0 +1,301 @@
...
+	currentValue, currentErr := sysutil.CgroupFileRead(cgroupTaskDir, file)
+	if currentErr != nil {
+		return currentErr
+	}
+	if strings.Contains(currentValue, value) {
+		klog.V(6).Infof("read before write %s %s, and contain same value %s", cgroupTaskDir,
+			file.ResourceFileName, value)
+		return nil
+	}
+	// 获取 value 的 maj:min 和 值
```
Comment (zwzhang0107) [inline pkg/koordlet/resmanager/types.go:289]: english translation

**Review by jasonliu747 — COMMENTED** [review]
Please add a document  about which linux kernel should user use if they wanna enable CPU burst.

File: pkg/koordlet/resmanager/cpu_burst.go (hunk 1 above)
Comment (hormes) [reply pkg/koordlet/resmanager/cpu_burst.go:1]: +1

**Pull request merged**